ironic-source-list.j2

```diff
@@ -0,0 +1,18 @@
+crudini
+gunicorn
+{% if env.IRONIC_SOURCE_HASH %}
+ironic @ git+https://opendev.org/openstack/ironic@{{ env.IRONIC_SOURCE_HASH }}
+{% elif env.IRONIC_SOURCE_GERRIT %}
+ironic @ git+https://opendev.org/openstack/ironic@{{ env.IRONIC_SOURCE_GERRIT }}
+{% else %}
+ironic @ git+https://opendev.org/openstack/ironic
+{% endif %}
+ironic-inspector @ git+https://opendev.org/openstack/ironic-inspector
```

Comment:
Will likely need the same treatment.

Reply:
yeah, in the end I'll try to cover as much packages/libraries as possible